pkg/build/controller/strategy/util.go

```diff
@@ -289,6 +289,15 @@ func addOutputEnvVars(buildOutput *corev1.ObjectReference, output *[]corev1.EnvV
 	return nil
 }
 
+// addTrustedCAMountEnvVar sets the BUILD_MOUNT_ETC_PKI_CATRUST environment variable if the build
+// pod needs the CA trust bundle (`/etc/pki/ca-trust`) mounted into build processes.
+func addTrustedCAMountEnvVar(mountTrustedCA *bool, output *[]corev1.EnvVar) {
```

Comment:
I think that the name `output` is confusing here, since it is not strictly an output, how about something like `envVars`?